mmv1/third_party/terraform/website/docs/r/logging_folder_bucket_config.html.markdown

```diff
@@ -28,6 +28,22 @@ resource "google_logging_folder_bucket_config" "basic" {
 }
 ```
 
+Create logging bucket with index configs
+
+```hcl
```

Comment:
Nit: spacing on these blocks. They should be 2-spaced spaces and are currently mixed tabs+spaces.

(All tabs would also work for consistency within the file, but the other entry in this file is also wrong)